docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -68,30 +68,40 @@ You are now signed in to Semgrep App.
 
 #### Permissions for GitHub
 
-This section explains why Semgrep App requires specific permissions. Semgrep App requires the following permissions in order to log in through GitHub:
+This section explains Semgrep App permissions. Semgrep App requests the following permissions when you log in through GitHub:
 
 <dl>
     <dt>Verify your GitHub identity</dt>
     <dd>Enables Semgrep App to read your GitHub profile data, such as your username.</dd>
     <dt>Know which resources you can access</dt>
-    <dd>Enables Semgrep App to access and display private repositories you can scan.</dd>
+    <dd>Enables Semgrep App to display private repositories you can scan.</dd>

```

Comment:
I don't think we have permission to do this when the user login is authorized. I think this permission comes from the GitHub app installation.

Reply:
Got it, removed it. :)